src/requests.jl

```diff
@@ -275,16 +275,8 @@ See also [`request_json`](@ref) and [`request_raw`](@ref).
         require_base_url = require_base_url,
     )::String
 
-    # Recall that the default log levels are:
-    #     Error === LogLevel(2_000)
-    #     Warn === LogLevel(1_000)
-    #     Info === LogLevel(0)
-    #     Debug === LogLevel(-1_000)
-    #
-    # Ref: https://docs.julialang.org/en/v1/stdlib/Logging
+    @debug "FHIRClient.request()" path verb tryparse_json(response_body)
```

Comment:
The old form was intentional. Even though I'm going with log level -1000 for now, I want us to explore other log levels going forward. By keeping the syntax as `LogLevel(-1_000)`, it's much easier to switch between different log levels in the future.

Reply:
OK, I reverted it.